src/gui/AutomationPatternView.cpp

```diff
@@ -278,6 +278,7 @@ void AutomationPatternView::paintEvent( QPaintEvent * )
 
 	const float y_scale = max - min;
 	const float h = ( height() - 2 * TCO_BORDER_WIDTH ) / y_scale;
+	const float ppTact  = ppt / MidiTime::ticksPerTact();
```

Comment:
It's the german/european word for a Bar in music: https://en.wikipedia.org/wiki/Bar_(music)